pkg/cedar/cedarldap/translated_client.go

```diff
@@ -94,3 +95,45 @@ func (c TranslatedClient) FetchUserInfo(ctx context.Context, euaID string) (*mod
 		EuaUserID:  person.UserName,
 	}, nil
 }
+
+// QueryPersons fetches a user's personal details by their common name
+func (c TranslatedClient) QueryPersons(ctx context.Context, commonName string) ([]*models2.UserInfo, error) {
```

Comment:
I checked the CEDAR API's behavior; it doesn't automatically do fuzzy searching. `commonName=Dylan` won't return anything, but `commonName=Dylan*` will. This code as written puts the burden of adding wildcards on the frontend; I think that for right now (given our use cases), we should make this method a fuzzy search that automatically puts a `*` on both ends of the given search string. We probably want to rename this method to something like `SearchPersonsByCommonNameFuzzy` or something, as well as possibly renaming the GQL query.

Reply:
ff9352132